src/nix/main.cc

```diff
@@ -79,8 +78,8 @@ struct NixArgs : virtual MultiCommand, virtual MixCommonArgs
         mkFlag()
             .longName("print-build-logs")
             .shortName('L')
-            .description("print full build logs on stderr")
-            .set(&printBuildLogs, true);
+            .description("print full build logs on stderr. DEPRECATED, use '--log-format bar-with-logs' instead")
```

Comment:
This message implies that `-L` is also deprecated, which should not be the case.

Reply:
Indeed. I've removed it altogether. Tell me if you still want a depreciation warning on `--print-build-logs`